napari/layers/utils/stack_utils.py

```diff
@@ -0,0 +1,188 @@
+import warnings
+from typing import Union, List, Tuple, Dict
+import itertools
+
+import numpy as np
+from napari.layers import Image, Labels
+from napari.utils.colormaps import colormaps
+from napari.layers.image._image_utils import guess_multiscale
+from napari.utils.misc import ensure_iterable, ensure_sequence_of_iterables
+
+
+def split_channels(
+    data: np.ndarray, channel_axis: int, **kwargs,
+) -> List[Tuple]:
+    """Function to split the data array into seperate arrays along an axis
+
+    Parameters
+    ----------
+    data : array or list of array
+    channel_axis : int
+        Axis to split the image along.
+    kwargs: keyword arguments
+
+    Returns
+    -------
+    List of LayerData tuples: [(data: array, metadata: Dict, type: str )]
+
+    """
+
+    # Determine if data is a multiscale
+    multiscale = kwargs['multiscale']
+    if multiscale is None:
+        multiscale, data = guess_multiscale(data)
+    kwargs['multiscale'] = multiscale
```

Comment:
```suggestion
    if 'multiscale' not in kwargs or kwargs['multiscale'] is None:
        multiscale, data = guess_multiscale(data)
        kwargs['multiscale'] = multiscale
```

Reply:
I'm suggesting this change so code will still run even if `multiscale` not in kwargs